form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/handler/FormFlowStepTypeFormHandler.kt

```diff
@@ -60,9 +60,8 @@ class FormFlowStepTypeFormHandler(
         additionalProperties: Map<String, Any>
     ) {
 
-        // TODO: Make sure the FormFlowInstance.additionalProperties contain the 'documentId' and the 'taskInstanceId'
         val documentId = additionalProperties["documentId"] as String?
-        val taskInstanceId = additionalProperties["taskInstanceId"] as String?
+        val taskInstanceId = additionalProperties["taskId"] as String?
```

Comment:
The FormFlowCreateTaskEventHandler shows that it should be 'taskInstanceId'

Reply:
you're right